### PR TITLE
mpa-8092-Configuration-service-automation-tests

### PR DESCRIPTION
**Pull request by PavloEsco**
Finished Create Repo API TC-s

### CODE REVIEW

File: scalecube-configuration/src/test/java/resources/features/mpa-8092-Configuration service - Automatic testing.feature [hunk 1]
```diff
@@ -1,122 +1,150 @@
...
-    Then any of repositories shouldn't be created and each of the user should get an error message: "role: "userId" not in role Owner"
-
+  #__________________________________________________NEGATIVE___________________________________________________________
 
-  #MPA-7103 (#1.2)
-  Scenario: Fail to create the Repo upon a "token" is invalid (expired)
-    Given a user have got an invalid "token" (API key)
-    When this user requested to create the "repository" with "specified" name
-    Then new "repository" shouldn't be created and the user should get an error message: "Token verification failed"
+  #MPA-8092 (#2)
```
Comment (ronenhamias) [inline scalecube-configuration/src/test/java/resources/features/mpa-8092-Configuration service - Automatic testing.feature]: this means it is implemented?

Comment (PavloEsco) [reply scalecube-configuration/src/test/java/resources/features/mpa-8092-Configuration service - Automatic testing.feature]: As you said before non of the tests wasn’t implemented.
So this file is not the exeption so each of these tests which are specified in this feture file not implemented.
I started ti update existing ones which I already pushed up to mpa-7103. 
But during the implementation Aharon closed his pull request under these tests i guess due to the spots in the business logic.

Comment (PavloPetrina) [reply scalecube-configuration/src/test/java/resources/features/mpa-8092-Configuration service - Automatic testing.feature]: As you said before non of the tests wasn’t implemented.
So this file is not the exeption so each of these tests which are specified in this feture file not implemented.
I started ti update existing ones which I already pushed up to mpa-7103. 
But during the implementation Aharon closed his pull request under these tests i guess due to the spots in the business logic.

Comment (PavloEsco) [reply scalecube-configuration/src/test/java/resources/features/mpa-8092-Configuration service - Automatic testing.feature]: I am going to proceed with tests and discuss them with the team, i guess potentially with Dmytro and Gabriel, I wanted to involve Assaf to the discussion also

**Pull request closed without merging**